interface/patient_file/letter.php

```diff
@@ -430,6 +438,7 @@ function insertAtCursor(myField, myValue) {
 <form method='post' action='letter.php' id="theform" name="theform">
 <input type="hidden" name="formaction" id="formaction" value="">
 <input type='hidden' name='form_pid' value='<?php echo $pid ?>' />
+<input type='hidden' name='token' value="<?php echo hash_hmac('sha256', '/letter.php.theform', $_SESSION['token']);?>" />
```

Comment:
Same here change name to `csrf_token` maybe a helper function to embed this will be a plus so you dont have to parse all these